### PR TITLE
Revise Microservice Transaction Sample

### PR DESCRIPTION
**Pull request by brfrn169**
This PR revises the Microservice Transaction Sample. I'll add inline comments for the details. Please take a look!

### CODE REVIEW

File: microservice-transaction-sample/README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Microservice Transaction Sample
 
-This is a sample application for Microservice Transaction that uses Two-phase Commit Transactions in ScalarDB.
+This tutorial describes how to create a sample application for Microservice Transaction that uses Two-phase Commit Transactions in ScalarDB.
```
Comment (brfrn169) [inline microservice-transaction-sample/README.md:3]: The other changes are mainly refinements in wording.

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

File: microservice-transaction-sample/README.md [hunk 2]
```diff
@@ -156,24 +162,54 @@ The `getOrder` of Order Service is for transaction #3, and The `getOrders` of Or
...
+```shell
+$ docker-compose up -d cassandra mysql
+```
+
+Please note that you need to wait around more than one minute for the containers to be fully started.
+
+### Load schema
+
+You then need to apply the schema with the following commands.
+Please download the schema tool `scalardb-schema-loader-<version>.jar` that can be found in [releases](https://github.com/scalar-labs/scalardb/releases) of ScalarDB.
```
Comment (brfrn169) [inline microservice-transaction-sample/README.md:180]:
```suggestion
You then need to apply the schema with the following commands.
To download the schema loader tool, `scalardb-schema-loader-<VERSION>.jar`, see the [Releases](https://github.com/scalar-labs/scalardb/releases) of ScalarDB and download the version that you want to use.
```

File: microservice-transaction-sample/README.md [hunk 3]
```diff
@@ -156,24 +162,54 @@ The `getOrder` of Order Service is for transaction #3, and The `getOrders` of Or
...
+
+For MySQL:
+
+```shell
+$ java -jar scalardb-schema-loader-<version>.jar --config database-mysql.properties --schema-file customer-service-schema.json
```
Comment (brfrn169) [inline microservice-transaction-sample/README.md:185]:
```suggestion
$ java -jar scalardb-schema-loader-<VERSION>.jar --config database-mysql.properties --schema-file customer-service-schema.json
```

File: microservice-transaction-sample/README.md [hunk 4]
```diff
@@ -156,24 +162,54 @@ The `getOrder` of Order Service is for transaction #3, and The `getOrders` of Or
...
+```
+
+For Cassandra:
+
+```shell
+$ java -jar scalardb-schema-loader-<version>.jar --config database-cassandra.properties --schema-file order-service-schema.json --coordinator
```
Comment (brfrn169) [inline microservice-transaction-sample/README.md:191]:
```suggestion
$ java -jar scalardb-schema-loader-<VERSION>.jar --config database-cassandra.properties --schema-file order-service-schema.json --coordinator
```

**Pull request merged**